pkg/test/framework/components/echo/config.go

```diff
@@ -346,8 +346,8 @@ func (c Config) IsVM() bool {
 	return c.DeployAsVM
 }
 
-func (c Config) IsDelta() bool {
-	// TODO this doesn't hold if delta is on by default
+func (c Config) IsSotw() bool {
+	// TODO this doesn't hold if delta is off by default
 	return len(c.Subsets) > 0 && c.Subsets[0].Annotations != nil && strings.Contains(c.Subsets[0].Annotations.Get(SidecarProxyConfig), "ISTIO_DELTA_XDS")
```

Comment:
does ISTIO_DELTA_XDS still apply

Reply:
This check is not very robust but it works for our tests; we only set ISTIO_DELTA_XDS=false in the test (never ISTIO_DELTA_XDS=true, since its the default), so if we see the string 'ISTIO_DELTA_XDS' it must be SotW

Reply:
super confusing now